test/apis/v1alpha1/api_validation_integration_test.go

```diff
@@ -31,24 +31,24 @@ var (
 	nameWithInvalidSize           = "abcdef-123456789-123456789-123456789-123456789-123456789-123456789-123456789"
 )
 
-var _ = Describe("Test networking v1alpha1 API validation", func() {
+var _ = Describe("Test networking fleetnetv1alpha1 API validation", func() {
```

Comment:
```suggestion
var _ = Describe("Test networking v1alpha1 API validation", func() {
```
This one should be v1alpha1